contracts/evmscript/EVMScriptRegistry.sol

```diff
@@ -82,7 +90,7 @@ contract EVMScriptRegistry is IEVMScriptRegistry, EVMScriptRegistryConstants, Ar
     function getScriptExecutor(bytes _script) public view returns (IEVMScriptExecutor) {
         uint256 id = _script.getSpecId();
 
-        if (id == 0 || id >= executors.length) {
+        if (id == 0 || id >= executorsNextIndex) {
```

Comment:
This bit is not _strictly_ necessary anymore, since `entry.enabled` can only be true if the executor exists.

Reply:
You are right. And we can save an `SSTORE` 💫